requirements.txt

```diff
@@ -5,3 +5,4 @@ cpsk
 pygeocoder
 brutal-ng==0.3.9
 imhdsk
+calendar_sk
```

Comment:
@mrshu yes, but you don't have to create pull request to change it. just update it on server